Sources/GRPC/AsyncAwaitSupport/AsyncBidirectionalStreamingCall.swift

```diff
@@ -38,6 +38,7 @@ public struct AsyncBidirectionalStreamingCall<
 
   /// The `Channel` used to transport messages for this RPC.
   public var subchannel: Channel {
+    // swiftformat:disable:next redundantGet
```

Comment:
It doesn't look like `swiftformat` knows about read-only effectful properties yet. Removing this (not redundant) `get` causes:

```
'async' call in a function that does not support concurrency
```

Reply:
OK, let's just leave these in place for now then.